### ISSUE TITLE
Some characters have .italic alts which are missing from rvrn rules (relatively quick fix)

### ISSUE THREAD

**Issue by arrowtype** [arrowtype/recursive#327]
The following characters have `.italic` alternates designed and (seemingly) compatible, yet are currently excluded from `rvrn` rules in the designspace:

```
ȁ ḍ ḏ ṡ ṣ ṥ ṧ ṩ ṹ ṻ ṟ ȑ ȓ ṛ ḯ ḝ ḉ ȅ ȇ ȉ ȋ ȕ ȗ ẏ ẓ ṃ ṅ ṇ ṉ ḡ ḥ
```

Here are their unicodes, which will be helpful in collecting their glyph names in order to create rvrn rules:

```
0x0201 0x1E09 0x1E1D 0x0205 0x0209 0x020B 0x0211 0x0213 0x0215 0x0217 0x1E8F 0x1E93 0x1E43 0x1E45 0x1E47 0x1E5B 0x1E21 0x1E0D 0x1E0F 0x1E61 0x1E63 0x1E65 0x1E67 0x1E69 0x1E5F 0x1E79 0x1E7B 0x1E2F
```

These simply need to be added into the `rules` of the designspace, and should then work well.

**Comment by arrowtype** [comment]
```Python
unicodesToFind = "ȁ ḍ ḏ ṡ ṣ ṥ ṧ ṩ ṹ ṻ ṟ ȑ ȓ ṛ ḯ ḝ ḉ ȅ ȇ ȉ ȋ ȕ ȗ ẏ ẓ ṃ ṅ ṇ ṉ ḡ ḥ".split()

unicodesOrdsToFind = []

for uni in unicodesToFind:
    unicodesOrdsToFind.append(ord(uni))

f = CurrentFont()

glyphUnicodes = {}

for g in f:
    try:
        glyphUnicodes[g.unicodes[0]] = g.name
    except:
        pass

for ordNum in unicodesOrdsToFind:
    if ordNum in glyphUnicodes.keys():
        print(f'<sub name="{glyphUnicodes[ordNum]}" with="{glyphUnicodes[ordNum]}.italic"/>')
```

**Comment by arrowtype** [comment]
Fixed with `be20892a8`!

![Kapture 2020-03-18 at 16 53 10](https://user-images.githubusercontent.com/45946693/77006523-f9e76d00-6938-11ea-8b0b-6ab9f0d1bb9d.gif)
